### PR TITLE
Automate census docker build

### PR DESCRIPTION
**Pull request by AdeHarrison**
# Motivation and Context
We want travis build to push docker images to Google Container Registry instead of dockerhub

# What has changed
Replaced docker image prefix with GCR registry location
NOTE: before merge, remove "automate-census-docker-build" branch condition and "branches only" section in .travis.yml file

# How to test?
From travis dashboard, switch to  "automate-census-docker-build" branch and "Restart Build". Once complete, check latest Docker image is in GCR.

# Links

[Trello](https://trello.com/c/dC6sEWqE/443-set-up-automated-docker-image-build)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,30 +6,29 @@ services:
...
-
 cache:
   directories:
   - $HOME/.m2
 
 branches:
   only:
-  - master
\ No newline at end of file
+  - automate-census-docker-build
```
Comment (davidmort) [inline .travis.yml]: shouldn't this be "rm-census"?

File: .travis.yml [hunk 2]
```diff
@@ -6,30 +6,29 @@ services:
...
+  - cp .maven.settings.xml $HOME/.m2/settings.xml
+  - mvn fmt:check
 
 script: travis_wait mvn verify cobertura:cobertura-integration-test
 
 after_success:
-- if [ "$TRAVIS_BRANCH" == "master" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]; then
-  docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD";
-  docker push sdcplatform/casesvc;
+- if ([ "$TRAVIS_BRANCH" == "rm-census" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ]) || ([ "$TRAVIS_BRANCH" == "automate-census-docker-build" ]); then
```
Comment (davidmort) [inline .travis.yml]: need to remove the second part of this "if" as per the PR description before it can be merged

File: .travis.yml [hunk 3]
```diff
@@ -6,30 +6,29 @@ services:
 language: java
 jdk: openjdk8
 
+env:
+  global:
+  - SOURCE_IMAGE_NAME="casesvc"
```
Comment (davidmort) [inline .travis.yml]: Is there any benefit the env vars given they're only used once?

File: .travis.yml (hunk 1 above)
Comment (AdeHarrison) [reply .travis.yml]: fixed

**Pull request merged**